apps/ledger-live-mobile/src/components/RootNavigator/NotificationCenterNavigator.tsx

```diff
@@ -57,12 +58,12 @@ export default function NotificationCenterNavigator() {
           title: t("notificationCenter.news.title"),
           headerRight: () => (
             <Flex flexDirection="row">
-              {incidents.length > 0 && (
+              {incidents.length >= 0 && (
```

Comment:
Should be `> 0` strict if we look at the specs :)

Reply:
Forgot to change it back after testing 😓